### PR TITLE
Fix mobile responsive UI issue and dark mode color issue

### PR DESCRIPTION
**Pull request by akash3444**
### Description

This PR fixes the mobile responsive issue on the landing page. Additionally, it corrects the color mismatch of input labels and error messages in dark mode.

### Changes Made

- Adjusted the CSS to ensure elements are properly scaled on smaller screens.
- Fixed the color of input labels in dark mode when there is an error.
- Corrected the color of error messages in dark mode.

### Screenshots

Before (Uneven horizontal padding):
Screenshot 2024-07-20 at 9 27 24 AM

After (Even horizontal padding):
Screenshot 2024-07-20 at 9 26 49 AM

Before:
Screenshot 2024-07-20 at 9 28 52 AM

After:
Screenshot 2024-07-20 at 9 29 15 AM

### Checklist

- [ ] I have tagged the issue in this PR.
- [x] I have attached necessary screenshots.
- [x] I have provided a short description of the PR.
- [x] I ran `yarn build` and build is successful
- [x] My code follows the style guidelines of this project.
- [ ] I have added necessary documentation (if applicable)

### CODE REVIEW

**Review by ICEPrey — APPROVED** [review]

**Comment by nang-dev** [comment]
nice! thanks

**Comment by nang-dev** [comment]
tysm

**Review by nang-dev — COMMENTED** [review]
approve - lol after merge oops

**Pull request merged**
